recipes/pipenv/meta.yaml

```diff
@@ -0,0 +1,56 @@
+  
+{% set name = "pipenv" %} 
+{% set version = "2018.11.26" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+
+# cuz source code didn't work, i used wheel instead.
```

Comment:
```suggestion
```

Reply:
i tried pipenv source to build before
it didnt work, it always showed error, so i change it to wheel
i will try source again

Reply:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=80344
it shows errors again now
i still dont know why
i think there are some bug in the source code

Reply:
If you scroll up to the first error it tells you there are missing dependencies:
```
    RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=https://pypi.org/simple/parver/
```
The solution is to add `parver` to the `host` dependencies.

Reply:
> please let me use wheel

It's very rare for wheels to be accepted for the reasons outlined [here](https://github.com/conda-forge/tensorflow-probability-feedstock/issues/3#issuecomment-462491626). Exceptions are extremely rare and only made after it's been shown building from source is too difficult. (tensorflow is now being built from source so even that exception no longer applies)

Reply:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=80348

it shows errors again
i cant figure why

Reply:
> > please let me use wheel
> 
> It's very rare for wheels to be accepted for the reasons outlined [here](https://github.com/conda-forge/tensorflow-probability-feedstock/issues/3#issuecomment-462491626). Exceptions are extremely rare and only made after it's been shown building from source is too difficult. (tensorflow is now being built from source so even that exception no longer applies)

i built pipenv in my channel before
pipenv-2018.5.18 can be built from source
after that i had to build from wheel
the dependencies are the same

i just dont know why